### PR TITLE
Add a new target KakuteH7

### PR DESCRIPTION
**Pull request by jamming**
KakuteH7 is the latest flight controllor model from holybro.  It  base on STM32H743 MCU. 
KakuteH7 provides 8 pwm output channels and on-board Bluetooth telemetry.

### CODE REVIEW

File: configs/default/HBRO-KAKUTEH7.config [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+serial 1 1 115200 57600 0 115200
+serial 5 64 115200 57600 0 115200
+serial 6 1024 115200 57600 0 115200
+
+# master
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set mag_i2c_address = 0
+set mag_spi_device = 0
+set mag_hardware = NONE
```
Comment (DusKing1) [inline configs/default/HBRO-KAKUTEH7.config:133]: If this setting is used, Betaflight will explicitly NOT support the use of compasses on this hardware and instruct users to buy a new board from a different manufacturer if they want to use a compass.
If you do not want this, please leave this parameter at its default (AUTO).

**Review by DusKing1 — CHANGES_REQUESTED** [review]

File: configs/default/HBRO-KAKUTEH7.config (hunk 1 above)
Comment (jamming) [reply configs/default/HBRO-KAKUTEH7.config]: Fix it now!

File: configs/default/HBRO-KAKUTEH7.config [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+dma pin D12 14
+# pin D12: DMA2 Stream 6 Request 29
+dma pin E09 12
+# pin E09: DMA2 Stream 4 Request 11
+
+# feature
+feature RX_SERIAL
+feature TELEMETRY
+feature OSD
+feature AIRMODE
```
Comment (mikeller) [inline configs/default/HBRO-KAKUTEH7.config:118]: Please remove all settings that are defaults - see the instructions at betaflight/betaflight@master/docs/TargetMaintenance/CreatingAUnifiedTarget.md for a way to do this.

**Review by mikeller — CHANGES_REQUESTED** [review]

File: configs/default/HBRO-KAKUTEH7.config [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+serial 1 1 115200 57600 0 115200
+serial 5 64 115200 57600 0 115200
+serial 6 1024 115200 57600 0 115200
+
+# master
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set mag_i2c_address = 0
+set mag_spi_device = 0
+set mag_hardware = AUTO
```
Comment (mikeller) [inline configs/default/HBRO-KAKUTEH7.config:128]: There are still plenty of default settings in here - please read **and follow** the instructions at https://betaflight/betaflight@master/docs/TargetMaintenance/CreatingAUnifiedTarget.md.
Also, as per instructions above, add `feature OSD` if your board has an on-board OSD.

**Review by mikeller — CHANGES_REQUESTED** [review]

**Comment by jamming** [comment]
sorry,
I use betaflight configurator 10.7.0, there is no STM32H743 unified-target here yet.
I will update it and fix the config file.
&nbsp;
&nbsp;
------------------&nbsp;Original&nbsp;------------------
From: ***@***.***&gt;;
Date: &nbsp;Tue, Aug 10, 2021 09:02 PM
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; ***@***.***&gt;; 
Subject: &nbsp;Re: [betaflight/unified-targets] Add a new target KakuteH7 (#491)

&nbsp;



 
@mikeller requested changes on this pull request.
 
 
In configs/default/HBRO-KAKUTEH7.config:
 &gt; + +# feature +feature TELEMETRY + +# serial +serial 0 1 115200 57600 0 115200 +serial 1 1 115200 57600 0 115200 +serial 5 64 115200 57600 0 115200 +serial 6 1024 115200 57600 0 115200 + +# master +set mag_bustype = I2C +set mag_i2c_device = 1 +set mag_i2c_address = 0 +set mag_spi_device = 0 +set mag_hardware = AUTO  
There are still plenty of default settings in here - please read and follow the instructions at ***@***.***/docs/TargetMaintenance/CreatingAUnifiedTarget.md.
 Also, as per instructions above, add feature OSD if your board has an on-board OSD.
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by mikeller** [comment]
@jamming: Yes there is - if you select 'Show unstable releases', and then 'Enable expert mode', and 'Development' as the build type:
image

**Review by mikeller — APPROVED** [review]

**Pull request merged**